piv/pcsc.go

```diff
@@ -48,16 +49,92 @@ func (e *scErr) Error() string {
 	return fmt.Sprintf("unknown pcsc return code 0x%08x", e.rc)
 }
 
+// AuthErr is an error indicating an authentication error occurred (wrong PIN or blocked).
+type AuthErr int
```

Comment:
nit: can this stay as a struct? don't know if there might be additional data we might want to add, but just to play it safe.

```go
type AuthErr struct {
    Retries int
}
```

Reply:
Sure.